deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -373,6 +373,10 @@ cephClusterSpec:
       osd:
         disabled: false
 
+  # Create an external rook-ceph cluster
```

Comment:
While I see now that this would work since it is under the `cephClusterSpec`, I worry that it will cause confusing since all the settings for a converged setting are still there. What if instead we added the following comment on ~line 75 immediately after the `cephClusterSpec:` line? And you could also add this as a paragraph in the cluster doc you modified above.
```
  # This cluster spec example is for a converged cluster where all the Ceph daemons are running locally,
  # as in the host-based example (cluster.yaml). For a different configuration such as a 
  # PVC-based cluster (cluster-on-pvc.yaml), external cluster (cluster-external.yaml),
  # or stretch cluster (cluster-stretched.yaml), replace this entire `cephClusterSpec`
  # with the specs from those examples. 
```

Reply:
Done